File: c/bee1101.c
#include <stdio.h>
 
int main() {
    while(1){
        int m, n, soma=0;
        scanf("%d %d", &m, &n);

        if(m==0 || m<0 || n==0 || n<0){
            break;
        }
        if(m<n){
            for(int i=m; i<=n; i++){
                printf("%d ", i);
                soma += i;
            }
            printf("Sum=%d\n", soma);
        } else if(m>n){
            for(int i=n; i<=m; i++){
                printf("%d ", i);
                soma += i;
            }
            printf("Sum=%d\n", soma);
        }
    }
    
    return 0;
}

File: python/bee1101.py
while True:
    m, n = map(int, input().split())
    soma = 0
    if m <= 0 or n <= 0:
        break
    if m <= n:
        for c in range(m, n+1):
            print(c, end=' ')
            soma += c
        print(f'Sum={soma}')
    else:
        for c in range(n, m+1):
            print(c, end=' ')
            soma += c
        print(f'Sum={soma}')
